packages/wallets/src/features/cashier/modules/Transfer/components/TransferFormAccountCard/TransferFormAccountCard.scss

```diff
@@ -7,12 +7,28 @@
     &--is-input {
         .wallets-transfer-form-account-card__icon {
             margin-right: 0.8rem;
+
+            .wallets-market-currency-icon {
+                margin-bottom: 0.4rem;
+
+                &__currency-icon {
+                    transform: translate(-0.3rem, 0.7rem);
+                }
+            }
         }
     }
 
     &--is-modal {
         .wallets-transfer-form-account-card__icon {
             margin-right: 1.6rem;
+
+            .wallets-market-currency-icon {
+                margin-bottom: 0.4rem;
+
+                &__currency-icon {
```

Comment:
@wojciech-deriv the transaction page is the only page that has this custom sizing. I've already provided a version with a smaller size however the layout differs from the rest. This is because, it has to accommodate the small size of the field.